tests/src/rules/dynamic-import-chunkname.js

```diff
@@ -1242,7 +1269,7 @@ context('TypeScript', () => {
         {
           code: `import(
             /* webpackChunkName: "someModule" */
-            /* webpackMode: "eager" */
+            /* webpackMode: "lazy" */
```

Comment:
This test would fail if it was eager mode, changing it to lazy so it's more correct.